src/main/java/com/sem/btrouble/controller/Controller.java

```diff
@@ -154,6 +156,29 @@ public void processInput(int delta) {
                 fireEvent(new PlayerEvent(p1, PlayerEvent.SHOOT, "Shot a rope"));
             }
         }
+        if (Model.getPlayers().size() > 1) {
+            secondPlayer(delta);
+        }
```

Comment:
Als secondPlayer in een aparte methode staat, kunnen de controls voor firstPlayer denk ik ook wel in een aparte methode. Misschien kan je zelfs een methode voor allebei de players maken waarbij je de player en keys als attributen doorgeeft. Als je wil, kan ik dat ook maken.